docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises-manual.md

```diff
@@ -62,13 +60,14 @@ Database servers need incoming TCP/IP access enabled to the following ports, for
 
 | Protocol | Port | Description |
 | :------- | :--- | :---------- |
-| TCP | 22 | SSH (for automatic administration) |
+| TCP | 22 | SSH (optional but recommended for troubleshooting) |
```

Comment:
This isn't optional, it's still required for configure steps to install YB software.